tracer/test/Datadog.Trace.Security.Unit.Tests/WafIpBlockTests.cs

```diff
@@ -39,13 +39,15 @@ public void TestOk()
             var readwriteLocker = new AppSec.Concurrency.ReaderWriterLock();
             using var context = waf.CreateContext(readwriteLocker);
             var result = context!.Run(new Dictionary<string, object> { { AddressesConstants.RequestClientIp, "51.222.158.205" } }, WafTests.TimeoutMicroSeconds);
-            result.ReturnCode.Should().Be(ReturnCode.Match);
-            result.Actions.Should().NotBeEmpty();
-            result.Actions.Should().Contain("block");
+            result.Should().NotBeNull();
+            result!.ReturnCode.Should().Be(ReturnCode.Match);
+            result!.Actions.Should().NotBeEmpty();
+            result!.Actions.Should().Contain("block");
```

Comment:
Just FYI, this is one of the reasons we generally don't bother setting `#nullable enable` in test projects: it tends to just add noise 🙂

Reply:
ok wont add in the others 😅